Content.Server/GameObjects/Components/PlaceableSurfaceComponent.cs

```diff
@@ -39,6 +75,8 @@ public override void ExposeData(ObjectSerializer serializer)
             base.ExposeData(serializer);
 
             serializer.DataField(ref _isPlaceable, "IsPlaceable", true);
+            serializer.DataField(ref _placeCentered, "PlaceCentered", false);
+            serializer.DataField(ref _positionOffset, "PositionOffset", Vector2.Zero);
```

Comment:
The original was probably done before style guide but it should be camel cased (placeCentered and positionOffset).

Reply:
Changed variables formatting to camel based for Client and Server files. Ty for the review btw